app/controllers/miq_ae_class_controller.rb

```diff
@@ -1955,11 +1958,14 @@ def create_action_url(node)
 
   def get_rec_name(rec)
     column = rec.display_name.blank? ? :name : :display_name
-    if rec.kind_of?(MiqAeNamespace) && rec.domain? && (!rec.editable? || !rec.enabled)
-      add_read_only_suffix(rec, rec.send(column))
-    else
-      rec.send(column)
+    if rec.kind_of?(MiqAeNamespace) && rec.domain?
+      editable_domain = Tenant.current_tenant.editable_domains.include?(rec)
+      enabled_domain  = Tenant.current_tenant.enabled_domains.include?(rec)
```

Comment:
@h-kataria @dclarizio @gmcculloug 
The enabled/disabled feature of a domain is not controlled per tenant. Only the owner can control if the domain is enabled or disabled. Do we have a use case where the domain would be enabled for some tenants and not for others?
Editable is a feature that changes from tenant to tenant.